packages/react-components/src/components/SendBoxErrors.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React, { useEffect } from 'react';
+import { SendBoxErrorBar } from './SendBoxErrorBar';
+
+/**
+ * @internal
+ * @TODO Add a timestamp to the error
+ */
+export type SendBoxError = {
+  message: string;
+};
+
+/**
+ * @internal
+ */
+export interface SendBoxErrorsProps {
+  fileUploadsPendingError?: SendBoxError;
+  onDismissFileUploadsPendingError?: () => void;
+  fileUploadError?: SendBoxError;
+}
+
+/**
+ * @internal
```

Comment:
if it's not used outside of the react-components package, mark as `@private`

Reply:
Thanks.